### PR TITLE
Updating prerequisites for RT

### PR DESCRIPTION
**Pull request by manishtr**
Signed-off-by: manish.r <manish.r@samsung.com>

### Change Description ###

Describe your changes here.

For instance,
 - Fix broken links of native application docs.
 -  ...

### Bugs Fixed ###

Provide links to bugs here

For instance,

 - Issue #265

### API Changes ###

In case of ACR, put the link of ACR

For instance,

 - ACR: ACR-1120

### CODE REVIEW

**Comment by mijin85cho** [comment]
Hello, @winstone77 
We have a suggestion about the place for the "Prerequisites for the TizenRT".
How about inserting the "Prerequisites for the TizenRT" at the /docs/application/tizen-studio/rt-ide/get-started/install.md?
We are thinking the red rectangle as the location where to add the prerequisites.

image

If you are going to move to here, please use 2 \#s, as like "## Prerequisites for the TizenRT" for the section title.

Would you please check the suggestion?

**Comment by sunghan-chang** [comment]
@mijin85cho Hello.
But it's the prerequisite of installing Tizen Studio. I think almost user does not read entire content before installing, it means that user might recognize above after installing Java12.

**Comment by mijin85cho** [comment]
@sunghan-chang I see. 
The reason we suggested to separate the page was, because for now it looks like two different pages (_prerequisites for Tizen Studio_ and _prerequisites for TizenRT_) are merged into one.

How about separating the "Prerequisites for TizenRT" at the below of "Prerequisites", at the red rectangle?

image


Or if want to keep those in one page, please move the location of the added context above the "Related Information" with titles with ##.
The result will be like below.


...(line 243)
\</tbody>
\</table> 

\## Prerequisites for the TizenRT

Check the following prerequisites before attempting to install the TizenRT.

\### Java Development Kit (JDK) Requirements

You must install Oracle Java Development Kit (JDK) 8.

Follow these instructions to install the appropriate JDK version for your system:

- Ubuntu

  Go to the \[Ubuntu Web site](https://help.ubuntu.com/community/Java) for detailed instructions for installing the Oracle&reg; JDK version 8. The raw binaries can be downloaded directly from Oracle (\[Oracle Java download page](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)).


\## Related Information
\* Dependencies
  \- Tizen Studio 1.0 and Higher


I actually not sure with this structure as it looks like "Prerequisites for the TizenRT" is added at the end of "Prerequisites for the Tizen Studio" as the sub section.
If possible, separating the context to another page would be better.
Please let me know your opinion about this.

**Comment by sunghan-chang** [comment]
@mijin85cho In my opinion, we had better leave it under description of java installation in `prerequisites of Tizen Studio`. 
```
- Ubuntu

  Go to the [Ubuntu Web site](https://help.ubuntu.com/community/Java) for detailed instructions for installing the Oracle&reg; JDK version 8. The raw binaries can be downloaded directly from Oracle ([Oracle Java download page](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)).

You can also use Oracle's OpenJDK. For more installation details, see [OpenJDK 12 and OpenJFX Installation Guide](openjdk.md).

> **NOTE** You must use Oracle Java Development Kit (JDK) 8 only to use RT IDE.
```

1. If we give two different ways to install the same thing, java, it could give confusion for user.
2. Almost user wants just following the sequence without knowing what it is and without checking entire content. Then, user could install java 12 before seeing TizenRT prerequisite.

**Comment by mijin85cho** [comment]
@sunghan-chang I think your suggestion is better than mine.
How about adding the note right after the ubuntu list item rather than after the last sentence?
It would become like below:

...

- Ubuntu

  Go to the [Ubuntu Web site](https://help.ubuntu.com/community/Java) for detailed instructions for installing the Oracle&reg; JDK version 8. The raw binaries can be downloaded directly from Oracle ([Oracle Java download page](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)).

  > **Note**
  >
  > For RT IDE use Oracle Java Development Kit (JDK) 8 only.

You can also use Oracle's OpenJDK. For more installation details, see [OpenJDK 12 and OpenJFX Installation Guide](openjdk.md).

## OS and System Requirements

...

**Comment by sunghan-chang** [comment]
@mijin85cho Yeah. No problem.

**Comment by mijin85cho** [comment]
Hello, @winstone77
As you can see @sunghan-chang and I think adding a note to the existing structure would be enough to achieve the goal of this PR.

Please check our suggestion and leave a comment if you have any other opinion.

Hello, @manishtr 
Would you please check the comments https://github.com/Samsung/tizen-docs/pull/937#issuecomment-555483779 and https://github.com/Samsung/tizen-docs/pull/937#issuecomment-555791013 to update the PR?

**Comment by manishtr** [comment]
> Hello, @winstone77
> As you can see @sunghan-chang and I think adding a note to the existing structure would be enough to achieve the goal of this PR.
> 
> Please check our suggestion and leave a comment if you have any other opinion.
> 
> Hello, @manishtr
> Would you please check the comments [#937 (comment)](https://github.com/Samsung/tizen-docs/pull/937#issuecomment-555483779) and [#937 (comment)](https://github.com/Samsung/tizen-docs/pull/937#issuecomment-555791013) to update the PR?

I have done 

> Hello, @winstone77
> As you can see @sunghan-chang and I think adding a note to the existing structure would be enough to achieve the goal of this PR.
> 
> Please check our suggestion and leave a comment if you have any other opinion.
> 
> Hello, @manishtr
> Would you please check the comments [#937 (comment)](https://github.com/Samsung/tizen-docs/pull/937#issuecomment-555483779) and [#937 (comment)](https://github.com/Samsung/tizen-docs/pull/937#issuecomment-555791013) to update the PR?

I have updated as per your comments

File: docs/application/tizen-studio/setup/prerequisites.md [hunk 1]
```diff
@@ -24,6 +24,8 @@ Follow these instructions to install the appropriate JDK version for your system
 
 You can also use Oracle's OpenJDK. For more installation details, see [OpenJDK 12 and OpenJFX Installation Guide](openjdk.md).
 
+> **NOTE** You must use Oracle Java Development Kit (JDK) 8 only to use RT IDE.
```
Comment (mijin85cho) [inline docs/application/tizen-studio/setup/prerequisites.md]: Please follow the note style in the comment. The indentation must be checked also.


  \> \*\*Note**
  \>
  \> For RT IDE use Oracle Java Development Kit (JDK) 8 only.

And this update should be placed above line 25, not after line 25.

Comment (manishtr) [reply docs/application/tizen-studio/setup/prerequisites.md]: done

**Pull request merged**